velox/tpch/gen/dbgen/include/tpch_constants.hpp

```diff
@@ -1,9 +1,26 @@
+/*
```

Comment:
@pedroerp : I was working with some of the files in the velox/tpch/gen/dbgen folder and there were some inconsistencies in the Facebook header placements. Some files (e.g. file https://github.com/facebookincubator/velox/blob/main/velox/tpch/gen/dbgen/include/dbgen/dss.h) had them, and some not. While working on this PR, as per instructions I did fix the headers I touched, but I wonder if the placement of the copyright is okay from a licensing perspective. 

I am keen to hear your take on it. wdyt ?